examples/tutorials/self-paced-training/part-3_security_and_privacy/chapter-6_Security_in_federated_compute_system/06.5_communition_security/communication_security.ipynb

```diff
@@ -0,0 +1,195 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Connection and Message Security\n",
+    "\n",
+    "Currently Flare's message security comes from mutual TLS: server and client authenticate each other when making direct connections. This means that only clients that have the right startup kits can make a connection to the server.\n",
+    "\n",
+    "However this solution may be hard for some customers due to their IT infrastructure policies. To enable customers to use their own connectivity solution, Flare now supports **BYOConn** (bring your own connectivity). \n",
+    "\n",
+    "Customer provided connectivity solution must meet the following requirements:\n",
+    "\n",
+    "* A client must be able to reach the server. The client and the server do not have to be connected directly. Any number of proxies can be used to provide the communication path between the client and the server.\n",
+    "* The communication path between the client and the server must ensure the confidentiality and integrity of messages.\n",
+    "\n",
+    "Since the connectivity solution may expose the server to the internet and allow any one to connect to the server, it is important to ensure that messages sent to the server are explicitly authenticated.\n",
+    "\n",
+    "Flare 2.5 and above support explicit message authentication: messages received by the server must have an auth token, and the token must be validated successfully to prove that it was issued by the server.\n",
```

Comment:
Do we need to emphasize this here? 

If we want, we could in the beginning (chapter 0) saying this tutorial is written for 2.5+